### PR TITLE
Update limitations to searching SITS in exo

### PR DESCRIPTION
**Pull request by StefanieBier**
Include teams. Remove ambiguity.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/keyword-queries-and-search-conditions.md [hunk 1]
```diff
@@ -156,7 +156,7 @@ For more information about creating queries using the `SensitiveType` property,
 
   Then you can use the ID in the `SensitiveType` search property to return documents that contain the custom sensitive data type; for example, `SensitiveType:7e13277e-6b04-3b68-94ed-1aeb9d47de37`
   
-- You can't use sensitive information types and the `SensitiveType` search property to search for sensitive data at-rest in Exchange Online mailboxes. However, you can use data loss prevention (DLP) policies to protect sensitive email data in transit. For more information, see [Overview of data loss prevention policies](data-loss-prevention-policies.md) and [Search for and find personal data](search-for-and-find-personal-data.md).
+- You can't use sensitive information types and the `SensitiveType` search property to search for sensitive data at-rest in Exchange Online mailboxes. This includes Teams chats and conversations as they are stored in the Exchange online mailbox. However, you can use data loss prevention (DLP) policies to protect sensitive email data in transit. For more information, see [Overview of data loss prevention policies](data-loss-prevention-policies.md) and [Search for and find personal data](search-for-and-find-personal-data.md).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/keyword-queries-and-search-conditions.md:159]:
```suggestion
- You can't use sensitive information types and the `SensitiveType` search property to search for sensitive data-at-rest in Exchange Online mailboxes. This includes Teams chats and conversations as they are stored in the Exchange Online mailboxes. However, you can use data loss prevention (DLP) policies to protect sensitive email data in transit. For more information, see [Overview of data loss prevention policies](data-loss-prevention-policies.md) and [Search for and find personal data](search-for-and-find-personal-data.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@StefanieBier

**Comment by markjjo** [comment]
Closed this PR because the same change was already made in PR#4353

**Pull request closed without merging**